contracts/satelites/Nutz.sol

```diff
@@ -80,14 +82,19 @@ contract Nutz is Ownable, ERC20 {
   // ########### ADMIN FUNCTIONS ################
   // ############################################
 
-  function powerDown(address powerAddr, address _holder, uint256 _amountBabz) onlyOwner {
-    // NTZ transfered from power pool to user's balance
-    Transfer(powerAddr, _holder, _amountBabz);
+  function powerDown(address powerAddr, address _holder, uint256 _amountBabz) public onlyOwner {
     bytes memory empty;
     _checkDestination(powerAddr, _holder, _amountBabz, empty);
+    // NTZ transfered from power pool to user's balance
+    Transfer(powerAddr, _holder, _amountBabz);
   }
 
 
+  function asyncSend(address _pullAddr, address _dest, uint256 _amountWei) public onlyOwner {
+    assert(_amountWei <= this.balance);
+    PullPayInterface(_pullAddr).asyncSend.value(_amountWei)(_dest);
```

Comment:
shouldn't we inherit `PullPayment` from `PullPayInterface` ? Like `contract PullPayment is Ownable, PullPayInterface`